### DISCUSSION TITLE
Persistent notification is missing browser notification feature?

### DISCUSSION THREAD

**Discussion by electrofloat** [home-assistant/frontend#11183] [Other feature requests]
I noticed this issue https://github.com/home-assistant/home-assistant-polymer/issues/3586 and got me thinking that is the https://developer.mozilla.org/en-US/docs/Web/API/notification supported?

Because when I get a persistent notification in Home Assistant there is no local browser notification (in firefox at least). Also the html5 push has browser notification but it is through push, not a local one.

**Comment by iantrich** [comment]
Seems like a good ide. Situations like this: https://github.com/home-assistant/home-assistant-polymer/issues/3082, could be a big annoyance.

Would have to see how often ha-sidebar is reloaded; might cause duplicate notifications to be created.

**Comment by electrofloat** [comment]
I see. Although the user could disable this anytime they want (also in the browser, not just in Home Assitant) and it shouldn't be on by default in this case I think.

On the plus side  the html5 notifications (or any other one) could use these to show local notifications instead of going through different servers to show push ones.
